fargate/upload-move/pkg/createClient.go

```diff
@@ -0,0 +1,37 @@
+package pkg
+
+import (
+	"context"
+	"errors"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	log "github.com/sirupsen/logrus"
+	"strings"
+)
+
+// CreateClient creates a client for the appropriate region data is being copied to
+func CreateClient(storageBucket string) (*s3.Client, string, error) {
+
+	region, exists := GetRegion(storageBucket)
+	if !exists {
+		return nil, "", errors.New("could not determine region code")
```

Comment:
might be helpful to have the bucket name in this error? 
```suggestion
		return nil, "", fmt.Errorf("could not determine region code from bucket name: %s", storageBucket)
```